Sources/UIApplication+handleSDLEvents.swift

```diff
@@ -250,27 +244,29 @@ public func onNativeTouch(
     let pressure: JavaFloat = try! jni.GetField("pressure", from: touchParameters)
     let timestampMs: JavaLong = try! jni.GetField("timestamp", from: touchParameters)
```

Comment:
FYI @michaelknoch if the current fix does work, then we can probably revert this code back to how it was as well (i.e. that Kotlin just sends these parameters directly, rather than sending an object that contains them)